tests/bin/rpminspect.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
+
+# Run RPMInspect on RPMs
+for f in build; do
```

Comment:
Is the loop supposed to iterate through each RPM file or just the `build` folder?

Reply:
Each file individually. Ideally you would do `rpminpspect-fedora <dir>` and it would check everything, but this doesn't allow the per-package exclusion of tests etc that we want.

I was attempting to get all the RPMs and the SRPM in one loop but it is getting messy/confusing. It is probably going to be better to have a loop `for f in build/RPMS/*; do` and then execute the SRPM separately `rpminspect-fedora build/SRPMS`